release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,51 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, to filter operator packages by channel, for example, `mirror.operators.catalog.packages.channels`, you had to specify the default channel for the package, even if you did not intend to use the packages from that channel. Based on this information, the resulting catalog is considered invalid if the `imageSetConfig` does not contain the default channel for the package.
+
+[WARNING]
+====
+Modifying the default channel should not be taken lightly as it could have an impact on the Operator's upgrade paths. The use of `defaultChannel` is the responsibility of the end user.
+====
+
+This update introduces the `defaultChannel` field in the `mirror.operators.catalog.packages` section. You can now select a default channel. This action enables `oc-mirror` to build a new catalog that defines the selected channel in the `defaultChannel` field as the default for the package. 
+
+If you are interested in a specific channel, such as `stable-4.14`, and not the default channel, you can add the required default channel to the `imageSetConfig` under the `defaultChannel` of the package. This action enables `oc-mirror` to build a new catalog with the defined default channel and override the previous default channel, for example, in a catalog containing `operator A` with multiple channels and `stable-4.15` as the default channel.
+
+.Before introducing the `defaultChannel` field
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages:
+    - name: operatorA
+      channels: 
+      - name: stable-4.14 # what the user needs
+      - name: stable-4.15 # needed as it is default channel, to avoid error
+----
+
+.After introducing the `defaultChannel` field
+[source,yaml]
+----
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+    packages:
+    - name: operatorA
+      defaultChannel: stable-4.14 <1>
+      channels:
+      - name: stable-4.14 # what the user needs
+----
+<1> Specify the required default channel.
+
+An initial check is added to ensure that if the `defaultChannel` field is set, you must also define it in the `channels` section of `ImageSetConfig`
```

Comment:
This will also be a part of oc-mirror docs.

Reply:
Ack.